package.json

```diff
@@ -3,12 +3,17 @@
   "version": "0.1.0",
   "main": "Gruntfile.js",
   "author": "YOUR NAME HERE",
```

Comment:
Might as well add an author here if you're updating this file anyway.

Reply:
Probably so, but I'll wait until we decide how and if we're going to be publishing this.